app/controllers/api/v1/bands_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::V1::BandsController < ApplicationController
+
+  def index
+    render json: Band.all
+  end
```

Comment:
add a space beneath this controller

Reply:
this code shouldnt be merged in because there is no corresponding react feature that I can see. BUT ill allow it to move you forward. Normally this would be a rejected PR though